win_build_output/midl/google_update/x64/google_update_idl_p.c

```diff
@@ -4982,7 +4982,7 @@ static const google_update_idl_MIDL_TYPE_FORMAT_STRING google_update_idl__MIDL_T
         {
 			NdrFcShort( 0x0 ),	/* 0 */
 /*  2 */	
-			0x11, 0xc,	/* FC_RP [alloced_on_stack] [simple_pointer] */
+			0x11, 0xc,	/* FC_RP [allocated_on_stack] [simple_pointer] */

```

Comment:
Dropping this file isn't a big deal.

If there are paths that should be skipped, it's easy for me to skip them, just let me know.